scripts/train.py

```diff
@@ -247,6 +244,32 @@ def create_and_link_model_artifact(
     return artifact
 
 
+def classifier_exists_in_wandb(
+    namespace: Namespace,
+    target_path: ModelPath,
```

Comment:
I'm not familiar with `ModelPath`, and why the below hardcodes `:latest`. This might be something that would've been useful for my code! Could you explain the benefit of using this rather than a raw path please?

Reply:
tbh i'm not super familiar with `ModelPath` objects either! I think i found myself putting together objects which looked a lot like this, went looking for other code which was doing something similar, saw folks using `ModelPath` and decided to use it here... but I'm still not sure what the best practices or substantial benefits are...

I believe @jesse-c added the `ModelPath` class - do you have any extra insight to add?